ui_framework/components/_variables.scss

```diff
@@ -80,3 +80,6 @@ $toolBarItsemSpacing: 10px;
 $globalFormControlHorizontalPadding: 12px;
 $globalFormControlPadding: 3px $globalFormControlHorizontalPadding 4px;
 $globalFormInputHeight: 30px;
+$globalFormFieldDefaultWidth: 180px;
+$globalFormFieldSmallWidth: 60px;
+$globalFormFieldLargeWidth: 400px;
```

Comment:
Might want to follow a pattern with your sizes. 

``` 
$globalFormFieldSmallWidth: $globalFormFieldDefaultWidth / 3;
$globalFormFieldLargeWidth: $globalFormFieldDefaultWidth * 2;
```

...etc. Just some sort of math so that when they sit next to each other they _look_ like they were planned.